docs/ops.html

```diff
@@ -4499,8 +4499,15 @@ <h4 class="anchor-heading"><a id="eligible_leader_replicas_upgrade" class="ancho
 
   <h4 class="anchor-heading"><a id="eligible_leader_replicas_tool" class="anchor-link"></a><a href="#eligible_leader_replicas_tool">Tool</a></h4>
   <p>The ELR fields can be checked through the API DescribeTopicPartitions. The admin client can fetch the ELR info by describing the topics.
-    Also note that, if <code>min.insync.replicas</code> is updated for a topic, the ELR field will be cleaned. If cluster default min ISR is updated,
-    all the ELR fields will be cleaned.</p>
+    Note that when the ELR feature is enabled:
+    <ul>
+      <li>The override of <code>min.insync.replicas</code> in broker-level will be removed.</li>
```

Comment:
`will be removed` -> `is removed`

Reply:
We probably also want to warn the users that if they previously only set min.insync.replicas at the broker level, once they enable ELR, this info will be lost and they need to reset that at the cluster level. 

Is the above correct @CalvinConfluent ?